packages/jsx/src/index.ts

```diff
@@ -251,7 +253,7 @@ export const reatomJsx = (ctx: Ctx, DOM: DomApis = globalThis.window) => {
           if (error) throw error
           unlink(element, un)
           element.appendChild(innerChild)
-        } else {
+        } else if (!isFalsy(child)) {
```

Comment:
This isn't a good idea, try to render `Atom<undefined | number>`, firstly `undefiend`, then toggle to `123`, I think it will not work. Add the test for this pls.

Reply:
Only static values ​​go into this block of code. Static values ​​can only be updated if the component is re-rendered.